client/src/components/Form/Elements/FormSelect.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+    <div>
+        <b-row align-v="center">
+            <b-col v-if="display == 'radio' || display == 'radiobutton'">
```

Comment:
Since there is only one child at a time, the flex-box wrapper here does nothing and can be removed.

```suggestion
        <div v-if="display === 'radio' || display === 'radiobutton'">
```

Reply:
This was separately handled in: #15016. Goal is to use this PR as a resource but spin off individual input field variations separately, using the `setup`-tag.